5.x/crud-fields.md

```diff
@@ -1635,6 +1635,41 @@ Input preview:
 
 > NOTE: if you are having trouble uploading big images, please check your php extensions **apcu** and/or **opcache**, users have reported some issues with these extensions when trying to upload very big images. REFS: https://github.com/Laravel-Backpack/CRUD/issues/3457
 
+<hr>
+
+<a name="phone"></a>
+### phone <span class="badge badge-pill badge-info">PRO</span>
+
+Show a telephone input and let the user choose a flag from dropdown.
+
+```php
+[   // phone
+    'name'  => 'phone', // db column for phone
+    'label' => 'Phone',
+    'type'  => 'phone',
+
+    // OPTIONALS
+    'initialCountry' => 'cl',
+    'separateDialCode' => true,
+    'nationalMode' => true,
+    'autoHideDialCode' => false,
+    'placeholderNumberType' => 'MOBILE',
+    // the above are some officially supported and tested options, provided by intlTelInput.js;
+    // there are more, we just haven't tested them; check out their docs - you can use any of them 
+    // inside the config attribute; take note that the directly defined attributes will override anything defined in the config;
```

Comment:
I've just suggested changing the merge order in https://github.com/Laravel-Backpack/PRO/pull/103/files so this could become:

```suggestion
    // inside the config attribute; take note that attributes defined in `config` will override any defaults or previously set values;
    ```